parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -139,29 +139,45 @@ downloadContainerdWasmShims() {
     for shim_version in $CONTAINERD_WASM_VERSIONS; do
         binary_version="$(echo "${shim_version}" | tr . -)"
         local containerd_wasm_filepath="/usr/local/bin"
-        local containerd_wasm_url="https://acs-mirror.azureedge.net/containerd-wasm-shims/${shim_version}/linux/amd64"
-        if [[ $(isARM64) == 1 ]]; then
-            containerd_wasm_url="https://acs-mirror.azureedge.net/containerd-wasm-shims/${shim_version}/linux/arm64"
-        fi
+        if ["$shim_version" == "v0.15.1" ]; then
```

Comment:
I'm realizing now that we don't have any content tests around installed wasm shims, probably a good idea to add a new test here to make sure all this install logic is working as expected - content tests are ran at VHD build time: https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/test/linux-vhd-content-test.sh

Reply:
Will do!